src/TestReactors.jl

```diff
@@ -608,15 +610,79 @@ end;
 
     inter,pinter = FragmentBasedReactiveFilmGrowthInterfaceConstantT(domainfilm,domainliq,interfacerxns);
 
-    react,y0,p = Reactor((domainfilm,domainliq),(y0film,y0liq),(0.0,0.1),(inter,),(pfilm,pliq,pinter));
```

Comment:
What are all these other changes here?

Reply:
Oops probably something went wrong when I rebased this. Will fix it.

Reply:
Fixed!